amoskeyz/Epic-mail
# fix(user): Bug fix for UI pages

#### What does this PR do?
Fix UI Pages

#### Description of Task to be completed?
-make Epic mail logo appear clearly on the background image
-update readme
-make ui pages responsive and attractive


#### How should this be manually tested?
-open your browser
-navigate to 'https://amoskeyz.github.io/Epic-mail/UI/index.html
- or navigate to 'https://amoskeyz.github.io/Epic-mail/UI/Sign-up.html
- you should be able to see the hand pointer on the sign-up, sign-in button
-Epic mail logo appears clearly because of  the changed color
-updated readme on github ripo

#### What are the relevant pivotal tracker stories?
<a href='https://www.pivotaltracker.com/story/show/164658720'>#164658720</a>
